I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/script.rpy", line 532, in script
    jump j_q3
ScriptError: could not find label 'j_q3'.

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/script.rpy", line 532, in script
    jump j_q3
  File "C:\Users\tooki\OneDrive - University of Waterloo\CHE 120\The Waterloo Games\renpy-8.3.3-sdk\renpy\ast.py", line 1699, in execute
    rv = renpy.game.script.lookup(target)
  File "C:\Users\tooki\OneDrive - University of Waterloo\CHE 120\The Waterloo Games\renpy-8.3.3-sdk\renpy\script.py", line 1103, in lookup
    raise ScriptError("could not find label '%s'." % str(original))
ScriptError: could not find label 'j_q3'.

Windows-10-10.0.22631 AMD64
Ren'Py 8.3.3.24111502
theWaterlooGames 1.0
Sun Nov 24 17:16:29 2024
